api/app.js

```diff
@@ -10,24 +10,61 @@ var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
 var coursesRouter = require('./routes/courses');
 var reviewsRouter = require('./routes/reviews');
+var authRouter = require('./routes/auth-routes');
+
+const cookieSession = require("cookie-session");
+const passport = require('passport');
+require("./google-setup");
+
+// CITATION: used https://medium.com/free-code-camp/how-to-set-up-twitter-oauth-using-passport-js-and-reactjs-9ffa6f49ef0
```

Comment:
I think this was already added to read me?

Reply:
Yes, but we still need to be adding citations to the local code as well. If you look at the syllabus here https://blogs.ubc.ca/cpsc436i2020s/syllabus/#sec-5-5 "Citations/acknowledgments should be in a clear section in your main README.md, in your license (if you have one!), and repeated locally where you used the code/help."